CPackSourceConfig.cmake

```diff
@@ -0,0 +1,81 @@
+# This file will be configured to contain variables for CPack. These variables
+# should be set in the CMake list file of the project before CPack module is
+# included. The list of available CPACK_xxx variables and their associated
+# documentation may be obtained using
+#  cpack --help-variable-list
```

Comment:
how does this file differ from `CPackConfig.cmake`?
do you use either of these files?